Makefile.js

```diff
@@ -622,7 +622,12 @@ target.gensite = function(prereleaseVersion) {
 
     // 3. Copy docs folder to a temporary directory
     echo("> Copying the docs folder (Step 3)");
-    cp("-rf", "docs/*", TEMP_DIR);
+    docFiles.forEach(filePath => {
+        cp("-rf", `docs/src${filePath}*`, `${TEMP_DIR}/${filePath}`);
```

Comment:
I'm getting `Error: cp: dest is not a directory (too many sources)` here.

``${TEMP_DIR}/${filePath}`` evaluates to `./tmp///rules/`, there are two extra slashes.